src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -1743,6 +1743,31 @@ public function get_tag() {
 		return strtoupper( $tag_name );
 	}
 
+	/**
+	 * Indicates if the currently matched tag contains the self-closing flag.
```

Comment:
Maybe 
```suggestion
	 * Indicates if the currently matched tag contains the <self-closing/> flag.
```

to make it a bit more illustrative? 🤔

Reply:
was trying to follow the explicit constraint not to use HTML tags in summaries and tags

> No HTML markup or Markdown of any kind should be used in the summary. If the text refers to an HTML element or tag, then it should be written as “image tag” or “img” element, not “<img>“. - [PHPDoc standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#summary)